### DISCUSSION TITLE
update 1.1.0 beta release for MacOS to R10888

### DISCUSSION THREAD

**Discussion by rich2014** [doublecmd/doublecmd#1201] [General]
@alexx2000 

hi, Alex

could you please update the 1.1.0 beta version of MacOS to R10888 ?

i just fixed an important bug of Cocoa in Lazarus, which is very helpful for TabControl.

rich

**Comment by alexx2000** [comment]
Hi, I think it is not correct to do it. Release version has permanent source code (v1.1.0 tag). All changes after it will be in the 1.1.1 version, which will be released soon.

Is it fixed in DC or Lazarus? If in Lazarus then I can rebuild v1.1.0 release with new Lazarus.

**Comment by rich2014** [comment]
fixed in Lazarus, rebuild please.

what I was thinking earlier was that the files for MacOS versions, in addition to `1.1.0`, also had the tag `10881`.

**Reply by alexx2000** [reply]
I rebuilt packages but I did not replace at SourceForge yet. Will it break `brew` cask?

**Reply by rich2014** [reply]
@alexx2000 

yes, there is a hash in the brew file.

after thinking about it, i still recommend building and uploading doublecmd-1.1.0-10888.cocoa.aarch64.dmg. and the brew file can be updated later.

because the tag 10888 in the file name indicates the version, I don't think it violates the principle you mentioned.

of course, it's up to your opinion.
